### PR TITLE
Display tweaks for sale artwork modules

### PR DESCRIPTION
**Pull request by ashfurrow**
This is a partial fix for https://github.com/artsy/auctions/issues/316 

- [x] Fix spacing.
- [x] Adjust font size.
- [x] Add paddle image to left of bid.
- [x] Don't show "0 bids", show starting bid.
- [x] Update snapshot images.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">There were <a href="https://eigen-ci.s3.amazonaws.com/4060/index.html">snapshot errors</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_circle">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by ashfurrow** [comment]
Nice, looking sharp /cc @katarinabatina 
simulator screen shot apr 19 2017 5 06 01 pm

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">No CHANGELOG changes made</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Artsy/Views/Auction/SaleArtworkViewModel.swift [hunk 1]
```diff
@@ -44,7 +49,7 @@ extension PublicComputedProperties {
         }
 
         let bidString = saleArtwork.highestOrStartingBidString()
-        if includeNumberOfBids {
+        if includeNumberOfBids && (saleArtwork.bidCount?.intValue ?? 0) > 0 {
```
Comment (orta) [inline Artsy/Views/Auction/SaleArtworkViewModel.swift]: weird programming semantics for basically if this exists and is greater than 0

**Comment by orta** [comment]
I'm good to go, but I did wonder if that bit of logic can be improved for readability

File: Artsy/Views/Auction/SaleArtworkViewModel.swift (hunk 1 above)
Comment (ashfurrow) [inline Artsy/Views/Auction/SaleArtworkViewModel.swift]: Yeah, it could definitely be improved.

**Comment by katarinabatina** [comment]
@ashfurrow looks great! A little hard for me to tell but is all the type the same color? If not it should be (the lot, the paddle, all the metadata). Looks like the bid value might be a different gray (which was apart of the former implementation)

**Comment by ashfurrow** [comment]
Thanks! I'll double-check the colours 👍

**Comment by ashfurrow** [comment]
Yup yup, you were right Katarina the grey was different. Looking better now:

simulator screen shot apr 20 2017 11 23 41 am

**Pull request merged**